web/grab_to_pan.js

```diff
@@ -20,7 +20,7 @@ const CSS_CLASS_GRAB = "grab-to-pan-grab";
 class GrabToPan {
   /**
    * Construct a GrabToPan instance for a given HTML element.
-   * @param {Element} options.element
+   * @param {{element: Element}}
```

Comment:
That's not a pattern we use anywhere in the code-base really.
It seems more correct to, just above, add
```javascript
/**
 * @typedef {Object} GrabToPanOptions
 * @property {HTMLElement} element
 */
```
and then write this as follows
```suggestion
   * @param {GrabToPanOptions} options
```